Jenkinsfile

```diff
@@ -368,10 +368,14 @@ def NetCoreTest(String nodeName, String targetFramework) {
 
               script += " --baasurl http://mongodb-realm:9090 --baasappid ${appId.trim()}"
               // see https://stackoverflow.com/a/53782505
-              sh """
+              sh '''
+                # see https://github.com/dotnet/sdk/issues/11108
```

Comment:
Can we move this comment outside the `sh` invocation and also expand on it a little so people don't have to go to open the linked issue to understand what it means.